dls-controls/malcolmjs
# Feature/Tweaks and fixes

## Description

- link spline split overlapping highlights
- attribute click show table first 
- help messages on blank centre pane (bad and no attribute)
- palette icon bigger (36px)
- snackbar on palette to tell to drag
- Tooltips on method buttons
- block title should be MRI
- make attribute details sizes %
- links flashing back in on delete
- filter nav list children for attributes with widgets
- error on block meta getting stuck in loading 
- attribute labels in block details no longer clickable
- acknowledge error button added to info pane

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- investigate above points

## Agile board tracking

connect to #424 
closes #424 
connect to #421 
closes #421
connect to #417
closes #417
